### ISSUE TITLE
[ttLib] [S_V_G_.py] SVG table implementation and XML dump rely on glyph IDs

### ISSUE THREAD

**Issue by justvanrossum** [fonttools/fonttools#1185]
That seems brittle. Should use glyph names like the rest of ttLib.

**Comment by behdad** [comment]
Right. Problem is, it's embedded in the XML text of the SVG. Ugh! I was sure I have had filed an issue about this, but cannot find it.

**Comment by behdad** [comment]
I don't think we're going to do surgery on the XML to convert to/from glyphnames...

**Comment by anthrotype** [comment]
btw, we do do some of that XML surgery when subsetting SVG table and have to remap id="glyph123" attributes:

https://github.com/fonttools/fonttools/blob/57c7792477d8f33e78b514d06103bb2fea9cd0f6/Lib/fontTools/subset/svg.py#L127-L156

though subsetting SVG adds an optional dependency on lxml; for dumping/loading SVG with ttx, it'd preferable we do not require that.
I'm ok if we don't implement this, but equally ok if anybody else volunteers to implement this.
